CanHazFunny/CanHazFunny.Tests/IJokeServiceTests.cs

```diff
@@ -0,0 +1,36 @@
+﻿using Moq;
+using Xunit;
+
+namespace CanHazFunny.Tests;
+
+
+public class IJokeServiceTests
+{
+    [Fact]
+    public void GetJoke_RetrieveAJock_Success()
+    {
+        var mock = new Mock<IJokeService>();
+        mock.Setup(x => x.GetJoke()).Returns("Here is my joke :)");
+        MockClass mockClass = new();
+        Assert.Equal(mock.Object.GetJoke(), mockClass.GetJoke());
```

Comment:
```suggestion
        #pragma warning disable xUnit2006 // Do not use invalid string equality check
        Assert.Equal<string>(mock.Object.GetJoke(), mockClass.GetJoke());
        #pragma warning restore xUnit2006 // Do not use invalid string equality check

```
Mark said to use non-generic version even for strings. This was said to me in teams chat and he said to disable it and use him as justification